DOWNLOAD_STATS.md

```diff
@@ -5,14 +5,30 @@
 | `@nolyfill/array.prototype.findlast` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.findlast.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.findlast) |
 | `@nolyfill/array.prototype.findlastindex` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.findlastindex.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.findlastindex) |
 | `@nolyfill/array.prototype.flat` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.flat.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.flat) |
 | `@nolyfill/array.prototype.flatmap` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.flatmap.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.flatmap) |
+| `@nolyfill/array.prototype.flatmap` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.flatmap.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.flatmap) |
+| `@nolyfill/array.prototype.foreach` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.foreach.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.foreach) |
+| `@nolyfill/array.prototype.join` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.join.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.join) |
+| `@nolyfill/array.prototype.keys` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.keys.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.keys) |
+| `@nolyfill/array.prototype.push` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.push.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.push) |
 | `@nolyfill/array.prototype.reduce` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.reduce.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.reduce) |
+| `@nolyfill/array.prototype.reduceright` | [![npm](https://img.shields.io/npm/dt/@nolyfill/array.prototype.reduceright.svg?style=flat-square&logo=npm&logoColor=white&label=total%20downloads&color=333)](https://www.npmjs.com/package/@nolyfill/array.prototype.reduceright) |
```

Comment:
Nobody uses `array.prototype.reduceright` on npm. It has 0 dependent and its weekly downloads are less than 20.